fnt

```diff
@@ -210,8 +217,13 @@ case "$1" in
 		else
 			echo "Skipping non-existant Packages cache. Consider running: ${BASH_SOURCE[0]} update" >&2
 		fi
-		curl -s "$GINDEX/ofl/" |grep "a href" |sed 's,.*">,google-,;s,/.*,,' |grep -v "\.\.$" | awk "/$2/"
-		curl -s "$GINDEX/apache/" |grep "a href" |sed 's,.*">,google-,;s,/.*,,' |grep -v "\.\.$" | awk "/$2/"
+		if [[ "$(uname -s)" == "OpenBSD" ]]; then
+			curl -s "$GINDEX/ofl/" |ggrep "a href" |gsed 's,.*">,google-,;s,/.*,,' |ggrep -v "\.\.$" | awk "/$2/"
+			curl -s "$GINDEX/apache/" |ggrep "a href" |gsed 's,.*">,google-,;s,/.*,,' |ggrep -v "\.\.$" | awk "/$2/"
+		else
+			curl -s "$GINDEX/ofl/" |grep "a href" |sed 's,.*">,google-,;s,/.*,,' |grep -v "\.\.$" | awk "/$2/"
+			curl -s "$GINDEX/apache/" |grep "a href" |sed 's,.*">,google-,;s,/.*,,' |grep -v "\.\.$" | awk "/$2/"
+		fi
```

Comment:
It is the above six lines that may or may not be necessary. As stated previously, this is because of OpenBSD using the older versions of grep and sed.